lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+path = require 'path'
+
+module.exports =
+  config:
+    executable:
+      type: 'string'
+      default: 'pylint'
+    rcFile:
+      type: 'string'
+      default: ''
+    messageFormat:
+        type: 'string'
+        default: '%i %m'
+        description:
+            'Format for Pylint messages where %m is the message, %i is the
+            numeric mesasge ID (e.g. W0613) and %s is the human-readable
+            message ID (e.g. unused-argument).'
+
+  activate: ->
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-pylint.executable',
+      (newValue) =>
+        @executable = newValue
+    @subscriptions.add atom.config.observe 'linter-pylint.rcFile',
+      (newValue) =>
+        @rcFile = newValue
+    @subscriptions.add atom.config.observe 'linter-pylint.messageFormat',
+      (newValue) =>
+        @messageFormat = newValue
+
+    @regex = '^(?<line>\\d+),(?<col>\\d+),\
+               (?<type>\\w+),\
+               (?<msg_id>\\w\\d+):(?<message>.*)$'
```

Comment:
`msg_id` capture here is unnecessary as the helper module's parse method just ignores non-recognized captures.